### PR TITLE
Feature: Social Login

### PR DESCRIPTION
**Pull request by swy641205**
detail....

### CODE REVIEW

File: file3.txt
Comment (katrinargr) [inline file3.txt:1]: Good

File: file3.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+hello
```
Comment (katrinargr) [inline file3.txt:1]: :D

File: file3.txt [hunk 2]
```diff
@@ -1 +1 @@
-hello
+Hello
```
Comment (katrinargr) [inline file3.txt:1]: GGGGGOOD

**Review by katrinargr — APPROVED** [review]

**Comment by swy641205** [comment]
OUO

**Pull request merged**